tests/performance/pioperformance_rearr.F90

```diff
@@ -12,7 +12,7 @@ program pioperformance_rearr
         pio_rearr_opt_t, pio_rearr_comm_p2p, pio_rearr_comm_coll,&
         pio_rearr_comm_fc_2d_disable, pio_rearr_comm_fc_1d_comp2io,&
         pio_rearr_comm_fc_1d_io2comp, pio_rearr_comm_fc_2d_enable,&
-        pio_rearr_comm_unlimited_pend_req
+        pio_rearr_comm_unlimited_pend_req, PIO_NOERR
```

Comment:
Although Fortran is not case-sensitive (most people now write
lower-case since this is more legible), other pio_types in this
file are written in lower-case. To be consistent, it is better
to use pio_noerr instead (throughout this file).

Reply:
I think we should use "ALL CAP"s for constants

Reply:
OK, then we can change constants in the existing Fortran code to use "ALL CAP" (maybe in a later code cleanup PR).